README.md

```diff
@@ -57,7 +57,7 @@ The SRA download functionality has been removed from the pipeline (`>=3.2`) and
 
 1. Install [`Nextflow`](https://www.nextflow.io/docs/latest/getstarted.html#installation) (`>=21.10.3`)
 
-2. Install any of [`Docker`](https://docs.docker.com/engine/installation/), [`Singularity`](https://www.sylabs.io/guides/3.0/user-guide/), [`Podman`](https://podman.io/), [`Shifter`](https://nersc.gitlab.io/development/shifter/how-to-use/) or [`Charliecloud`](https://hpc.github.io/charliecloud/) for full pipeline reproducibility *(please only use [`Conda`](https://conda.io/miniconda.html) as a last resort; see [docs](https://nf-co.re/usage/configuration#basic-configuration-profiles))*. Note: This pipeline does not currently support running with Conda on macOS if the `--remove_ribo_rna` parameter is used because the latest version of the SortMeRNA package is not available for this platform.
+2. Install any of [`Docker`](https://docs.docker.com/engine/installation/), [`Singularity`](https://www.sylabs.io/guides/3.0/user-guide/) (you can follow [this tutorial](https://singularity-tutorial.github.io/01-installation/)), [`Podman`](https://podman.io/), [`Shifter`](https://nersc.gitlab.io/development/shifter/how-to-use/) or [`Charliecloud`](https://hpc.github.io/charliecloud/) for full pipeline reproducibility *(you can use [`Conda`](https://conda.io/miniconda.html) to install NextFlow but please only use it as a last resort for running the pipeline; see [docs](https://nf-co.re/usage/configuration#basic-configuration-profiles))*. Note: This pipeline does not currently support running with Conda on macOS if the `--remove_ribo_rna` parameter is used because the latest version of the SortMeRNA package is not available for this platform.
```

Comment:
```suggestion
2. Install any of [`Docker`](https://docs.docker.com/engine/installation/), [`Singularity`](https://singularity-tutorial.github.io/01-installation/), [`Podman`](https://podman.io/), [`Shifter`](https://nersc.gitlab.io/development/shifter/how-to-use/) or [`Charliecloud`](https://hpc.github.io/charliecloud/) for full pipeline reproducibility *(you can use [`Conda`](https://conda.io/miniconda.html) to install NextFlow but please only use it as a last resort for running the pipeline; see [docs](https://nf-co.re/usage/configuration#basic-configuration-profiles))*. Note: This pipeline does not currently support running with Conda on macOS if the `--remove_ribo_rna` parameter is used because the latest version of the SortMeRNA package is not available for this platform.
```

Reply:
I think it's fine to replace the install link directly because the links for Docker etc point to install pages too. Thanks!

Reply:
Those tutorial pages are hosted on the tools' official websites, which is not the case for this Singularity tutorial. Shouldn't we let the link to Sylabs' official website?